core/math/camera_matrix.cpp

```diff
@@ -123,6 +123,60 @@ void CameraMatrix::set_perspective(real_t p_fovy_degrees, real_t p_aspect, real_
 	*this = *this * cm;
 }
 
+void CameraMatrix::set_oblique(real_t p_fovy_degrees, real_t p_aspect, real_t p_z_near, real_t p_z_far, Quat p_oblique_plane, bool p_flip_fov) {
```

Comment:
CameraMatrix changes aren't related to editor grid changes.

Reply:
yes... this is for my ObliqueSpatialCamera PR... I'm not even sure how this made it into this PR...
my git log only shows edits to plugin_editor_spatial.h/cpp
I will have to hunt this down and remove it.

Thank you for finding it!

Reply:
okay.. not sure how this happened... But my commit for this PR somehow got rebased on top of two of my other feature commits. (I keep 1 per branch and merge them for my custom editor branch).

I will correct this before I reopen this PR for 3.x and master